packages/expo-cli/src/commands/initAsync.ts

```diff
@@ -361,37 +361,38 @@ async function installNodeDependenciesAsync(projectRoot: string, packageManager:
   }
 }
 
-export async function initGitRepoAsync(
-  root: string,
-  flags: { silent: boolean; commit: boolean } = { silent: false, commit: true }
-) {
+async function initGitRepoAsync(root: string) {
   // let's see if we're in a git tree
   try {
     await spawnAsync('git', ['rev-parse', '--is-inside-work-tree'], {
       cwd: root,
     });
-    !flags.silent && Log.log('New project is already inside of a git repo, skipping git init.');
-  } catch (e) {
+    // Log a light notice if we're in a git tree.
+    Log.log(
+      chalk.gray(`Project is already inside of a git repo, skipping ${chalk.bold`git init`}.`)
+    );
+    // Bail out if inside git repo, this makes monorepos a bit easier to setup.
+    return true;
+  } catch (e: any) {
     if (e.errno === 'ENOENT') {
-      !flags.silent && Log.warn('Unable to initialize git repo. `git` not in PATH.');
+      Log.warn('Unable to initialize git repo. `git` not in PATH.');
       return false;
     }
   }
 
   // not in git tree, so let's init
   try {
     await spawnAsync('git', ['init'], { cwd: root });
-    !flags.silent && Log.log('Initialized a git repository.');
-
-    if (flags.commit) {
-      await spawnAsync('git', ['add', '--all'], { cwd: root, stdio: 'ignore' });
-      await spawnAsync('git', ['commit', '-m', 'Created a new Expo app'], {
-        cwd: root,
-        stdio: 'ignore',
-      });
-    }
+    Log.debug('Initialized a git repository.');
+
+    await spawnAsync('git', ['add', '--all'], { cwd: root, stdio: 'ignore' });
```

Comment:
If you are already in a repo (in case of a monorepo), this part gets skipped, but wouldn't it still be nice to commit the changes `expo init` makes (albeit to the 'parent' repo)?